test-.tf

```diff
@@ -0,0 +1,11 @@
+resource "azurerm_postgresql_server" "example_ssl_enforcement_enabled" {
```

Comment:
:exclamation:**Cycode**: Infrastructure configuration issue: 'Enable PostgreSQL geo-redundant backups'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |

 <!-- 76b5a6c4-17e8-4979-8926-af049498c901 -->